oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/endpoint/OAuth2AuthorizationResponse.java

```diff
@@ -122,6 +133,7 @@ public static Builder error(String errorCode) {
 		private String errorCode;
 		private String errorDescription;
 		private String errorUri;
+		private MultiValueMap<String, String> parameters = new LinkedMultiValueMap<>();
```

Comment:
This is not needed. Please see previous comment on how to build the `OAuth2AuthorizationResponse.redirectUri` with query parameters.